fdbserver/DataDistribution.actor.cpp

```diff
@@ -4009,16 +4009,69 @@ ACTOR Future<std::vector<std::pair<StorageServerInterface, ProcessClass>>> getSe
 // to a sorted PID set maintained by the data distributor. If now no storage server exists, the new Process ID is 0.
 ACTOR Future<Void> updateNextWigglingStoragePID(DDTeamCollection* teamCollection) {
 	state ReadYourWritesTransaction tr(teamCollection->cx);
-	state Value writeValue;
+	state Value writeValue = ""_sr;
 	state const Key writeKey =
 	    wigglingStorageServerKey.withSuffix(teamCollection->primary ? "/primary"_sr : "/remote"_sr);
```

Comment:
This is not showing as a change in this PR, but I am curious if there is a HA failover, i.e., will the new primary read the old remote key, and vice versa? Should we use the DC id as the prefix?

Reply:
There are actually two keys \xff/storageWigglePID/primary and \xff/storageWigglePID/remote. Each DC would read their own respective keys to do the wiggle process.
For suppose, if there is an HA failover, the new primary would be reading the old remote key and vice versa. But, the processId in the key would not exist, so it cannot do any wiggling process. So, it tries to read the next process and now it gets the correct processId and the wiggle process continues.

This same would happen if there is a recover of the DC in non-HA scenario as the processId would change.